orchagent/main.cpp

```diff
@@ -157,6 +159,27 @@ void init_gearbox_phys(DBConnector *applDb)
     delete tmpGearboxTable;
 }
 
+void init_poe(DBConnector *applDb)
+{
+    Table tmpPoeTable(applDb, "_POE_TABLE");
+    PoeConfig poe(tmpPoeTable);
+
+    SWSS_LOG_NOTICE("POE: Initialize");
+    if (!poe.isPoeEnabled()) {
+        SWSS_LOG_WARN("POE: not enabled");
```

Comment:
added an additional check, so that if the platform does not not support PoE (doesn't have a poe_config.json file present) then we do not print anything